.github/workflows/dependabot_review.yaml

```diff
@@ -0,0 +1,57 @@
+name: Auto-Assign Reviewers and Run pnpm changeset for Dependabot PRs
+
+on:
+  pull_request_target:
+    types: [opened, synchronize]
+
+jobs:
+  run-changeset:
+    if: github.actor == 'dependabot[bot]'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: Install pnpm
+        run: npm install -g pnpm
+
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Create changeset file
```

Comment:
I don't think we should do it that way (too hacky). There must be a way to utilize `pnpm changeset` on the root directory to create a patch.